test/extended/cli/set-image.go

```diff
@@ -0,0 +1,128 @@
+package cli
+
+import (
+	"os"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-cli] oc set image", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		deploymentConfig = exutil.FixturePath("testdata", "cmd", "test", "cmd", "testdata", "test-deployment-config.yaml")
+		imageStream      = exutil.FixturePath("testdata", "cmd", "test", "cmd", "testdata", "image-streams", "image-streams-centos7.json")
```

Comment:
This is out of scope of this PR but I'd like to note that one day we'll need to switch different image-streams json file other than centos7.

Reply:
yes we do

Reply:
Some of those images are not centos7 based already :wink: but I guess the filename will stay the same